_pages/Dashboard/index.js

```diff
@@ -1,5 +1,33 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
 const Dashboard = () => {
-  return <h1>Dashboard</h1>;
+  const [player, setPlayer] = useState([]);
+
+  const getPlayerInfo = async () => {
+    const response = await axios.get('/api/identity');
+    setPlayer(response.data);
+  };
+
+  useEffect(() => {
+    getPlayerInfo();
+  }, []);
+
+  console.log(player);
+
+  return (
+    <div className="player-container" style={{ color: 'red' }}>
```

Comment:
A good class naming rule of thumb – and what we generally do at ClearScore – is to match the top-level class name with the component name.

For example:
```
const Dashboard = () => <div className="dashboard">...</div>;
const PlayerList = () => <div className="player-list">...</div>;
```

P.s. I know this came from my janky example in Slack 😂  Do as I say, not as I do...

Reply:
Nah it's fine, I've changed it. I was being a bit lazy when I wrote it because I wanted it to work